components/_patterns/02-molecules/date-bars/date-bar~ended.yml

```diff
@@ -0,0 +1,7 @@
+event_date_start:
+  "2015-12-17 12:00:00"
+event_date_end:
+  "2015-12-17 14:00:00"
+date_bar_modifiers:
+  1:
+    ended
```

Comment:
I don't think we should rely on someone manually (or having to do the logic in the presenter template). We can set this in the design template with a conditional check for whether the `event_date_end` time has passed, and if so, set the modifier automatically.

(Also, there should be a trailing new line at the end of all files :) you should be able to set your editor to do that automatically on save)

Reply:
Updated my editor, thank you. Also updated this code based on your suggestion, so going to resolve this comment.